### ISSUE TITLE
Error in Edge browser

### ISSUE THREAD

**Issue by PBarnard** [auth0-samples/auth0-angular-samples#155]
Just pulled the angular sample app and am getting an error in Edge browser on Windows 10. Works fine in Chrome.

Steps to reproduce:

- Pull repo
- Run npm install
- Create auth_config.json in application root
- Run ng serve
- Browse to localhost:4200
- Click the "Log In" button

![image](https://user-images.githubusercontent.com/14309231/60496352-87e7ce80-9caa-11e9-9c5a-27bc7590e91c.png)

Edit: Works fine in the Edge Chromium dev-channel preview version

**Comment by PBarnard** [comment]
Found a workaround, there's a polyfill that's deprecated by the owner but still works here:

https://github.com/inexorabletash/text-encoding

I cloned that, added the two .js files in the /lib folder to the angular app's src/ folder and added the following to angular.json:

```
"scripts": [
    "src/encoding-indexes.js",
    "src/encoding.js"
]
```

Appears to work fine in both Edge and Chrome now.

**Comment by stevehobbsdev** [comment]
Thanks @PBarnard, good to know there is a workaround 👍 

Anything we can do here @luisrudge?

**Comment by stevehobbsdev** [comment]
Hey @PBarnard, off the back of this issue we've just released v1.0.2 of the SDK which should fix this without having to use the workaround. Can you give this a try?

**Comment by PBarnard** [comment]
Hey @stevehobbsdev yeah that's working fine now in both browsers, cheers! 👍
